### PR TITLE
Shipment to pypi

### PR DESCRIPTION
**Pull request by sedgewickmm18**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,10 +2,34 @@
...
+    url='https://github.com/ibm-watson-iot/iotfunctions',
+    packages=find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+    ],
+    python_requires='>=3.6',
+    install_requires=requirements,
+    extras_require={'kafka': ['confluent-kafka==1.0.0']}
```
Comment (herleraja) [inline setup.py:34]: can be moved to requirment file itself.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,3 @@
+include mmfunctions/*.py
```
Comment (herleraja) [inline MANIFEST.in:1]: may I know the reason behind this function line?

**Review by herleraja — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (sedgewickmm18) [reply setup.py:34]: do you mean the kafka requirement ?

File: MANIFEST.in (hunk 2 above)
Comment (sedgewickmm18) [reply MANIFEST.in:1]: this is a bug. thanks for pointing it out

**Review by rsivaibm — APPROVED** [review]

**Pull request merged**